Source/WTF/wtf/SingleThreadIntegralWrapper.h

```diff
@@ -40,6 +40,8 @@ class SingleThreadIntegralWrapper {
     SingleThreadIntegralWrapper& operator++();
     SingleThreadIntegralWrapper& operator--();
 
+    IntegralType uncheckedGet() const { return m_value; }
```

Comment:
It's rather confusing to call this "uncheckedGet". What is unchecked in what sense isn't clear specially when it's used with CheckedPtr. But why do we need this variant to begin with?
Why can we not check the thread identity?

Reply:
I kind of agree that this is confusing. Not sure how to improve it.

I could change ptrCount() to have a function argument or template parameter `enum class DoThreadCheck { No, Yes }`. Or I could rename uncheckedPtrCount() to ptrCountWithoutThreadCheck(). Does one of those options seem better to you?

Reply:
Renaming it to `ptrCountWithoutThreadCheck` sounds good to me.